.gitignore

```diff
@@ -3,5 +3,5 @@
 .vscode/
 build/
 bin/
-.tmp/
+data/
```

Comment:
I would keep this dir for other purposes. But it's on your decision

Reply:
that was request from devops team, let's make their life a bit easier :)